src/components/AvatarGroup/AvatarGroup.module.scss

```diff
@@ -0,0 +1,66 @@
+@import "src/components/Avatar/Avatar.scss";
+@import "../../styles/themes";
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  position: relative;
+}
+
+.avatarContainer {
+  margin-left: -8px;
+  // TODO customize spacing
+  :hover {
+    z-index: 2;
+  }
+}
+
+.counterContainer {
+  background: var(--primary-background-hover-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: -8px;
+  z-index: 1;
+
+  &:focus {
+    outline: none;
+    @include focus-style(50%);
+  }
+
+  &.small {
+    height: $avatar-size-small;
```

Comment:
let's create a css mixin for it and use it here (for prevent a copy paste of the same code)